utils/utilities.js

```diff
@@ -265,5 +256,34 @@ module.exports = {
       return web3.eth.abi.encodeFunctionCall({ name, inputs, type: "function" }, params);
     }
     throw new Error(`Invalid method "${method}"`);
+  },
+
+  encodeTransaction: (to, value, data, isTokenCall = false) => ({ to, value, data, isTokenCall }),
+
+  addTrustedContact: async (wallet, target, module, securityPeriod) => {
+    const owner = await wallet.owner();
+    await module.addToWhitelist(wallet.address, target, { from: owner });
+    await utilities.increaseTime(securityPeriod + 2);
+    const isTrusted = await module.isWhitelisted(wallet.address, target);
+    assert.isTrue(isTrusted, "should be trusted after the security period");
+  },
+
+  // set the relayer nonce to > 0
+  initNonce: async (wallet, account, module, manager, securityPeriod) => {
+    await utilities.addTrustedContact(wallet, account, module, securityPeriod);
```

Comment:
Can we simplify this helper and assume `account` is always `account[8]` for example, similarly with how we default the relayer account to account[9]. It makes no difference but keeps the function interface simpler.